### PR TITLE
Merge whitelist contracts/methods changes from staging to production

### PR DESCRIPTION
**Pull request by commoddity**

### CODE REVIEW

File: .github/workflows/staging-us-west-2.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```
Comment (bvanderlugt) [inline .github/workflows/staging-us-west-2.yml:17]: I would probably do this in a seperate pr so if you rollback you don't change the build definition

File: .github/workflows/staging-us-west-2.yml [hunk 2]
```diff
@@ -27,9 +27,6 @@ jobs:
         id: login-ecr
         uses: aws-actions/amazon-ecr-login@v1
 
-      - name: Log the branch name
-        run: echo "Building image from branch ${{ github.ref }}""
```
Comment (bvanderlugt) [inline .github/workflows/staging-us-west-2.yml:31]: Is this for debugging or do you want it to run for every job on master?

Comment (commoddity) [reply .github/workflows/staging-us-west-2.yml:31]: It was for debugging; it's removed now.

File: .github/workflows/staging-us-west-2.yml (hunk 1 above)
Comment (commoddity) [reply .github/workflows/staging-us-west-2.yml:17]: Sure, I can do that.

File: cache/cache.go [hunk 3]
```diff
@@ -246,20 +248,84 @@ func (c *Cache) addGatewayAAT(aat repository.GatewayAAT) {
...
+			appSettings.WhitelistContracts[i].Contracts = inContract.Contracts
+			chainIDExists = true
+			break
+		}
+	}
+	if !chainIDExists {
+		appSettings.WhitelistContracts = append(appSettings.WhitelistContracts, inContract)
+	}
+
+	c.rwMutex.Lock()
```
Comment (adshmh) [inline cache/cache.go:296]: Thanks for making the change.

**Review by adshmh — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**